### PR TITLE
Conform with new chainId format

### PR DESCRIPTION
**Pull request by oed**
According to new chainId format.

### CODE REVIEW

File: CIPs/CIP-94/CIP-94.md [hunk 1]
```diff
@@ -21,14 +21,14 @@ A DID that uses this method MUST begin with the following prefix: `did:nft`. Per
 
 ## Method Specific Identifier
 
-The method specific identifier is simply a [CAIP-19 Asset ID](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-19.md) where all `:` characters have been replaced by `.` and all `/` characters replaced by `_`, in order to comply with the DID spec.
+The method specific identifier is simply a [CAIP-19 Asset ID](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-19.md) where all `/` characters is replaced by `_`, in order to comply with the DID spec.
```
Comment (stbrody) [inline CIPs/CIP-94/CIP-94.md:24]:
```suggestion
The method specific identifier is simply a [CAIP-19 Asset ID](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-19.md) where all `/` characters are replaced by `_`, in order to comply with the DID spec.
```

**Review by stbrody — APPROVED** [review]
lgtm mod question about why we replaced colons previously but no longer do

**Comment by oed** [comment]
Colons were replaced before because though they were not allowed, but turns out I misread the did-core spec.

**Comment by oed** [comment]
@ukstv Want to make sure you see this since it requires a change to the nft-did-resolver.

**Review by ukstv — APPROVED** [review]

**Pull request merged**